### DISCUSSION TITLE
Logs form Celery worker not show details.

### DISCUSSION THREAD

**Discussion by falukelo** [apache/airflow#37419] [Q&A]
Hello,
i'm deployed Airflow Cluster but when task error logs not show details.
<img width="783" alt="Screenshot 2567-02-14 at 18 32 25" src="https://github.com/apache/airflow/assets/38658824/d18ea1df-00f1-46ae-8baa-4e2cb567d548">

I always have to shell in and check the service logs to figure out what's going on.
<img width="925" alt="Screenshot 2567-02-14 at 18 37 33" src="https://github.com/apache/airflow/assets/38658824/e1977ff5-81f7-4c28-a3d8-1e0a8c030a8a">

**Comment by potiuk** [comment]
Task error logs have chance to show when the task has a chance to start. In this case you have wrong environment. Make sure that the same packages are installed when you parse a DAG and when you execute it. If you do no give airflow a chance to start because your environment is broken, you also do not give it a chance to log anything because it needs to start to be able to report anything.
